prompb/types.proto

```diff
@@ -127,6 +147,7 @@ message TimeSeries {
   repeated Sample samples       = 2 [(gogoproto.nullable) = false];
   repeated Exemplar exemplars   = 3 [(gogoproto.nullable) = false];
   repeated Histogram histograms = 4 [(gogoproto.nullable) = false];
+  Metadata metadata             = 5 [(gogoproto.nullable) = false];
```

Comment:
Initially I thought we wouldn't have to explicitly change the protocol since it was an extra optional signal (again like exemplars and histograms), if we think that's the way to go I'm up for it!

@cstyan also had some ideas about things that could go into a future v1.1 or v1.2 version of the protocol so we can discuss if we want to also bring any more changes.

Reply:
*Update*: Yeah, I'm also convinced this is the way to go! What would be the process to start things for drafting v1.1?

Getting consensus on the Google doc proposal and then opening a PR to the docs? Sending to the mailing list?